test/test.getch.ts

```diff
@@ -133,14 +133,27 @@ describe('🥁 configuration options', () => {
     scope.done();
   });
 
+  it('should merge URL parameters with params option', async () => {
```

Comment:
Could you add a test for the case where the url has a querystring, but no params are provided as well?

Reply:
Done!